CanalTP/Chaos
# Filter impacts with severities

# Description

This PR enables to filter impacts with severity IDs

## Issue (optional)

Issue link: BO-401

## Pull Request type (optional)

This is a new feature

## How to test

Here are the following steps to test this pull request:

- launch an post request on ``` /impacts/_search ``` end point with body like ```{"severity_ids": ["First_severity_UUID"], "Second_severity_UUID","ec"}```
- Retrieved impacts list should contain only impacts with required severity IDs

## Team reviewers (optional)

Mention here the people who should review your PR
